plugins/storage/volume/linstor/src/main/java/com/cloud/hypervisor/kvm/storage/LinstorStorageAdaptor.java

```diff
@@ -88,7 +88,7 @@ private String getHostname() {
             return result.trim();
         } catch (IOException | InterruptedException exc) {
```

Comment:
```suggestion
        } catch (Exception exc) {
```

if so, remove IOException from import.